.gitignore

```diff
@@ -11,3 +11,4 @@ test_results/*
 *.core
```

Comment:
In the commit message: `Firecracker compilation time` should be `cpu-template-helper compilation time`

Reply:
yeah, what I meant was Firecracker is the whole project including all the binaries (firecracker, jailer, etc.). But it would be better to be more precise. will fix. thanks.